crates/kitsune_p2p/kitsune_p2p/src/lib.rs

```diff
@@ -23,11 +23,10 @@ mod spawn;
 pub use spawn::*;
 
 #[allow(missing_docs)]
-#[cfg(feature = "test_utils")]
+#[cfg(any(test, feature = "test_utils"))]
 pub mod test_util;
 
-#[cfg(test)]
-#[cfg(feature = "test_utils")]
+#[cfg(any(test, feature = "test_utils"))]
```

Comment:
do we need the tests built when the `test_utils` feature is passed in? that would mean something external depends on the tests themselves, which i'd find awkard as it's a private module.

i suggest trying whether this suffices:

```suggestion
#[cfg(test)]
```

Reply:
@steveeJ I thought so too, but it seems correct as it is, because while the module will be compiled, all of the tests are still still `#[cfg(test)]`, because they are tests, so they won't be compiled just by having the feature flag turned on. However, all other functions will be compiled, which is what we do want.